src/biogeochem/CNAllocationMod.F90

```diff
@@ -503,7 +505,7 @@ subroutine calc_allometry(num_soilp, filter_soilp, &
        else
           g1 = grperc(ivt(p))
        end if
-       cnl  = leafcn(ivt(p))
+       cnl  = leafcn_t_evolving(p)
```

Comment:
In the original (2022) implementation, either I missed this change or that version of the code didn't need it.

Reply:
Looking back before my 2025 updates at ctsm5.1.dev074-9-g0fe849e20, I have confirmed that we did not have a separate CNAllocationMod file, so I do not think that I missed anything in the original implementation.